lib/buttons-and-links/__tests__/KButton.spec.js

```diff
@@ -67,9 +73,94 @@ describe('KButton', () => {
   });
 
   describe.visual('KButton Visual Tests', () => {
-    it('renders correctly with default props', async () => {
-      await renderComponent('KButton', { text: 'Test Button' });
-      await takeSnapshot('KButton - Default');
+    it('renders correctly with different appearances', async () => {
```

Comment:
IMO each snapshot should be a test case. Unit test cases should be simple and test just one thing. If we want to group multiple test cases like for example with the disable state, we could use a describe block inside de discribe.visual block and group them.

Reply:
So, if I am getting it correctly, for the case where we are testing different appearances of KButton for example, we need to write them all in separate `it` blocks and group those it blocks in one `describe` block?

Reply:
Yes, although Im not sure if we should group them as "different appearances", in general all snapshots are different appearances.

Reply:
Oh different appearances means different values for the `appearance` prop. I hadnt get it, in that case its up to you if it makes sense to group them, you can group them, if not its fine too.